pages/about.html

```diff
@@ -0,0 +1,122 @@
+            href="https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services"
+            target="_blank" class="usa-link usa-link--external">Technology Transformation Services</a>.
+        </div>
+      </div>
+      <div class="grid-col-12 desktop:grid-col-6 mobile-top-padding">
+        <h2 id="technology">With what technology is Data.gov built?</h2>
+        <div>
+          Data.gov is powered by two open source applications, <a href="https://ckan.org" target="_blank"
+            class="usa-link usa-link--external">CKAN</a> and
+          <a href="https://www.11ty.dev/" target="_blank" class="usa-link usa-link--external">11ty</a> and it is
+          developed publicly on <a href="https://github.com/GSA/catalog-deploy" target="_blank"
+            class="usa-link usa-link--external">GitHub</a>.
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<div class="grid-container">
+  <section class="usa-section">
+    <h2>Get Data on Data.gov</h2>
+    <h4>How are the datasets on Data.gov collected?</h4>
+    <div class="grid-row grid-gap-lg padding-top-2">
+      <div class="grid-col-12 desktop:grid-col-6 ">
+        Under the OPEN Government Data Act, which is Title II of the <a
+          href="https://www.congress.gov/115/plaws/publ435/PLAW-115publ435.pdf" target="_blank"
+          class="usa-link usa-link--external">Foundations for Evidence-Based Policymaking
+          Act</a>, government data is required to be made
+        available in open, machine-readable formats, while continuing to ensure privacy and security.
+
+        Government data publishers looking to get their data on Data.gov should read the detailed guide: <a
+          href="https://resources.data.gov/tools/how-to-get-your-open-data-on-datagov/" target="_blank"
+          class="usa-link usa-link--external">How to
+          get your open data on Data.gov</a>. The Data.gov team typically works with a designated open data point of
+        contact as a liaison for each agency. Data publishers should consult with their agency point of contact to
+        include any additional datasets on Data.gov. If you need help determining who your open data point of contact
+        is, please <a href="{% page '/contact/'%}">contact us</a>.
+
+      </div>
+      <div class="grid-col-12 desktop:grid-col-6 mobile-top-padding">
+        <b>Federal agencies are required to:</b>
+        <ul>
+          <li>Create a Single Agency Data Inventory.** Agencies are required to catalog their data assets, just like
+            they would inventory computers or desk chairs, to better manage and use these resources.</li>
+          <li>Publish a Public Data Listing.** Agencies are required to publish a list of their data assets that are
+            public, or could be made public. This list is made available as a data.json hosted at the primary domain
```

Comment:
```suggestion
          <li><b>Create a Single Agency Data Inventory.</b> Agencies are required to catalog their data assets, just like
            they would inventory computers or desk chairs, to better manage and use these resources.</li>
          <li><b>Publish a Public Data Listing.</b> Agencies are required to publish a list of their data assets that are
            public, or could be made public. This list is made available as a data.json hosted at the primary domain
```

I wasn't sure if the `**` was pointing to an endnote or was supposed to be part of bolding or a separator element... but the most logical answer is to just remove it.